README.md

```diff
@@ -1,140 +1,142 @@
-2. __Yarn__ installieren, falls noch nicht vorhanden ([Anleitung](https://yarnpkg.com/en/docs/install)).
-3. Im Terminal in unser Verzeichnis wechseln und mittels Yarn die benötigten Pakete holen.  
-_Achtung, es kommen mehrere hundert Megabyte an Daten durch die Leitung!_  
+1. __Node__ (>= 6.9) install, if not already available. Can be downloaded as a package directly from the website: [https://nodejs.org](https://nodejs.org)
+2. __Yarn__ install, if not already available ([Manual](https://yarnpkg.com/en/docs/install)).
+3. Change to our directory in the terminal and get the needed packages by Yarn.
+_Attention, several hundred megabytes of data are coming through the line!_  
 
         $ yarn
 
-4. __Gulp__, falls noch nicht vorhanden, mittels Yarn global installieren:
+4. __Gulp__, if its not installed, you can install it with Yarn global:
 
         $ yarn global add gulp-cli
 
-5. Einen __localhost/vhost__ (z. B. `http://local.bimmelbam`) für unser Projekt anlegen, der für REDAXO funktioniert. Zielverzeichnis des Hosts ist `/app`!  
-_Anm.: Eine gute Anleitung gibt es hier: [»Eine lokale Domain mittels Apache Virtual Hosts erstellen«](http://www.matthias-zeis.com/ressourcen/zfstde/zfbook.creating.a.local.domain.using.apache.virtual.hosts.html)_
-6. Die `.env` (__Environment__) im Projektroot anlegen. Dazu einfach die im Paket enthaltene `.env.example` kopieren und entsprechend anpassen: `APP_HOST` entspricht dem eben angelegten Host, `APP_ENV` belassen wir vorerst auf `development`.
-7. Den Frontend-Workflow starten. Danach sollte unser Frontend-Prototyp erreichbar sein über `http://localhost:3000/demo/`:
+5. Create a __localhost / vhost__ (for example, `http: // local.bimmelbam`) for our project that works for REDAXO. Target directory of the host is `/app`!  
+_Btw: Here you find a good Guide: [»Create a local domain using Apache Virtual Hosts«](http://www.matthias-zeis.com/ressourcen/zfstde/zfbook.creating.a.local.domain.using.apache.virtual.hosts.html)_
+6. Create the `.env` (__Environment__) in the projectroot. Just copy the `.env.example` included in the package and adjust it accordingly: `APP_HOST` corresponds to the newly created host, `APP_ENV` we leave for the time being on `development`.
+7. Start the front-end workflow. After that, our frontend prototype should be accessible via `http://localhost:3000/demo/`:
 
         $ gulp
 
-Hat nicht geklappt? Sorry :-(  
-Nicht ärgern und lieber im [REDAXO-Slack](http://redaxo.org/slack/) nachfragen oder im [Forum](http://www.redaxo.org/de/forum/allgemeines-f39/frontend-workflow-fur-redaxo-mit-gulp-browserify-postcss-t21541.html#p120663), dort wird dir geholfen!
+Did not work? Sorry :-(  
+Don't worry!. you can ask the [REDAXO-Slack](http://redaxo.org/slack/) or in the [Forum](http://www.redaxo.org/de/forum/allgemeines-f39/frontend-workflow-fur-redaxo-mit-gulp-browserify-postcss-t21541.html#p120663), here you will get help!
```

Comment:
Don’t worry! (Or even better: No worries!)
Punctuation: the correct typographic apostrophe `’` is entered via `option` + `shift` + `#` (on Mac). Personally, I like it more than the single quotation mark `'` (`shift` + `#` on Mac). But you decide.